.teamcity/src/subprojects/release/publishing/PublishMavenBuild.kt

```diff
@@ -0,0 +1,29 @@
+package subprojects.release.publishing
+
+import jetbrains.buildServer.configs.kotlin.v10.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildType
+import subprojects.build.core.*
+import java.lang.RuntimeException
+
+class PublishMavenBuild(private val build: Build) : BuildType({
+    id("KtorPublishMavenBuild_${build.name}".toExtId())
+    name = "Publish ${build.name} to Maven"
+
+    dependencies {
+        val buildId = build.build?.id
+        if (buildId != null) {
+            artifacts(buildId) {
+                buildRule = lastSuccessful()
+                artifactRules = stripReportArtifacts(artifactRules)
+            }
+        } else {
+            throw RuntimeException("Build ID not found for entry ${build.name}")
```

Comment:
You just throw an exception early and omit else branch for readability.

Reply:
Sure, but it's just a different style.